string status             # DISACTIVATED, PENDING , IN PROGRESS or FAILED
string current_command